__tests__/test.js

```diff
@@ -246,6 +246,22 @@ describe('prettyFormat()', () => {
     expect(prettyFormat(val)).toEqual('Object {\n  "prop1": Object {},\n  "prop2": Object {},\n}')
   });
 
+  it('prints an HTMLElement', () => {
+    const val = document.createElement('div');
+
+    val.className = 'foo';
+    val.id = 'bar';
+
+    val.appendChild(document.createElement('span'));
+
+    const pretty = prettyFormat(val);
+
+    expect(pretty).toContain('HTMLDivElement');
+    expect(pretty).toContain('"className": "foo"');
+    expect(pretty).toContain('"id": "bar"');
+    expect(pretty).toContain('[HTMLSpanElement]');
```

Comment:
Rather than asserting the individual pieces can you assert the entire thing? Whitespace is often an issue